scripts/start.js

```diff
@@ -268,13 +284,24 @@ detect(DEFAULT_PORT).then(port => {
   }
 
   clearConsole();
-  var question =
-    chalk.yellow('Something is already running on port ' + DEFAULT_PORT + '.') +
-    '\n\nWould you like to run the app on another port instead?';
 
-  prompt(question, true).then(shouldChangePort => {
-    if (shouldChangePort) {
-      run(port);
-    }
-  });
+
+  getExistingProcesses(DEFAULT_PORT)
+    .then(res => chalk.yellow(
+      `Something is already running on port ${DEFAULT_PORT}:
+
+      The existing process is: ${res}
+
+      Would you like to run the app on another port instead?
+      `
+    ))
+    .then(question => prompt(question, true))
+    .then((shouldChangePort) => {
+      if (shouldChangePort) {
+        run(port);
+      }
+    })
+    .catch((err) => {
+      throw new Error(err);
```

Comment:
We don't want to abort the entire thing if we couldn't determine what's running on the port. Not printing the hint around what is running on that port would be better.